modules/metering-install-configure-storage.adoc

```diff
@@ -0,0 +1,215 @@
+// Module included in the following assemblies:
+//
+// * metering/metering-installing-metering.adoc
+
+[id="metering-install-configure-storage_{context}"] 
+= Configuring Metering storage
+
+Configure the metering operator by editing the default metering configuration file created when you install the metering stack. Some configuration options, such as storage, must be configured before installing the metering stack. Use the documentation below and make changes to the `spec` section of the default configuration file.
+
+[NOTE]
+====
+The Configuring Metering documentation covers all available
```

Comment:
I'm not sure I understand the purpose of this note, or what the "Configuring Metering documentation" is.